src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -262,6 +264,8 @@ export const ExpressCheckoutUI = ({ order }: ExpressCheckoutUIProps) => {
         variables: {
           input: {
             id: orderData.internalID,
+            paymentMethod: "CREDIT_CARD",
+            creditCardWalletType: "APPLE_PAY",
```

Comment:
We need to set payment method related fields on the order. This sets it on confirming Apple Pay, but we can move it earlier if makes sense.

Reply:
The unused `expressPaymentType` variable above has this:
```
export type ExpressPaymentType =
  | 'google_pay'
  | 'apple_pay'
  | 'amazon_pay'
  | 'link'
  | 'paypal';
```